### ISSUE TITLE
clang format linter doesn't show clear errors messages

### ISSUE THREAD

**Issue by simonppg** [github/super-linter#2036]
The default configuration of clang format linter spots errors but no clear description is given.

```shell
2021-10-05 17:44:35 [INFO]   File:[/github/workspace/app/src/main/cpp/linux/LinuxFilesManager.cpp]
2021-10-05 17:44:35 [ERROR]   Found errors in [clang-format] linter!
2021-10-05 17:44:35 [ERROR]   Error code: 1. Command output:
------
/github/workspace/app/src/main/cpp/linux/LinuxFilesManager.cpp:11:62: error: code should be clang-formatted [-Wclang-format-violations]
LinuxFilesManager::LinuxFilesManager(const char *projectPath)
                                                             ^
------
2021-10-05 17:44:35 [INFO]   ---------------------------
2021-10-05 17:44:35 [INFO]   File:[/github/workspace/app/src/main/cpp/linux/LinuxFilesManager.hpp]
2021-10-05 17:44:35 [ERROR]   Found errors in [clang-format] linter!
2021-10-05 17:44:35 [ERROR]   Error code: 1. Command output:
------
/github/workspace/app/src/main/cpp/linux/LinuxFilesManager.hpp:16:7: error: code should be clang-formatted [-Wclang-format-violations]
#endif  // APP_SRC_MAIN_CPP_LINUX_LINUXFILESMANAGER_HPP_
      ^
------
2021-10-05 17:44:35 [INFO]   ---------------------------
2021-10-05 17:44:35 [INFO]   File:[/github/workspace/app/src/main/cpp/linux/LinuxPlatform.cpp]
2021-10-05 17:44:35 [INFO]    - File:[LinuxPlatform.cpp] was linted with [clang-format] successfully
2021-10-05 17:44:35 [INFO]   ---------------------------
2021-10-05 17:44:35 [INFO]   File:[/github/workspace/app/src/main/cpp/linux/LinuxPlatform.hpp]
2021-10-05 17:44:35 [ERROR]   Found errors in [clang-format] linter!
2021-10-05 17:44:35 [ERROR]   Error code: 1. Command output:
------
/github/workspace/app/src/main/cpp/linux/LinuxPlatform.hpp:26:7: error: code should be clang-formatted [-Wclang-format-violations]
#endif  // APP_SRC_MAIN_CPP_LINUX_LINUXPLATFORM_HPP_
      ^
------
```
